app/assets/javascripts/OutdoorMap/map_build.js

```diff
@@ -80,20 +80,32 @@ for (const feature of buildings) {
     layers[feature.properties.campus].addLayer(layer);
 }
 
+//Add points of interest
+layers['Points of Interest'] = L.layerGroup().addTo(mymap);
+let pois = JSON.parse(document.getElementById('container').dataset.source);
+for(const feature of pois) {
```

Comment:
I would rename the variable feature to interestedPoint or similar, as feature would rather translate to Funktion, Eigenschaft oder Merkmal (for me the context does not quite fit)